libass/ass.h

```diff
@@ -388,6 +388,16 @@ void ass_renderer_done(ASS_Renderer *priv);
  */
 void ass_set_frame_size(ASS_Renderer *priv, int w, int h);
 
+/**
+ * \brief Set the number of threads to use during rendering. Default is 0,
+ * meaning the number of cores if known, otherwise 1.
```

Comment:
Nit: Given `_ONLN` takes e.g.g cgroup limits into account and may not exclude usable cores which just need to be woken up again, how about phrasing this as  *“meaning the number of available logical cores if known”*?